aten/src/ATen/native/mps/OperationUtils.h

```diff
@@ -1,5 +1,7 @@
 //  Copyright © 2022 Apple Inc.
 
+#pragma once
```

Comment:
Guards possible duplicate includes in the future.

Reply:
Looks good.